website/docs/search.md

```diff
@@ -75,7 +75,12 @@ module.exports = {
     // ...
     // highlight-start
     algolia: {
+      // If Algolia did not provide you any appId, use 'BH4D9OD16A'
+      appId: 'YOUR_APP_ID',
+
+      // Public API key: it is safe to commit it
       apiKey: 'YOUR_API_KEY',
```

Comment:
```suggestion
      apiKey: 'YOUR_SEARCH_API_KEY',
```

Not related to this PR, but we can avoid confusion between write/admin/search API keys